test/test_operations.py

```diff
@@ -853,6 +853,9 @@ def __init__(self, x, y):
     nested = Nested(b, c)
     self.assertRaises(RuntimeError, lambda: xm.check_view_sharing(nested))
 
+    tmpfile = tempfile.mktemp()
+    self.assertRaises(RuntimeError, lambda: torch.save([b, c], tmpfile))
```

Comment:
This leaves the file around?

Reply:
Hmmmm this API doesn't create new file so it's fine. 
https://docs.python.org/2/library/tempfile.html#tempfile.mktemp 
But given the warning in doc I might use another API instead :P